src/native/managed/cdacreader/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/StackWalk/FrameHandling/IPlatformFrameHandler.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Diagnostics.DataContractReader.Contracts.StackWalkHelpers;
+
+namespace Microsoft.Diagnostics.DataContractReader.Contracts.StackWalkHelpers;
+
+/// <summary>
+/// Interface for handling platform-specific frames.
+/// Methods return true if the context was updated from the Frame. False otherwise.
+/// </summary>
+internal interface IPlatformFrameHandler
+{
+    public abstract bool HandleInlinedCallFrame(Data.InlinedCallFrame frame);
```

Comment:
afaik all these 'public' and 'abstract' modifiers do nothing since these methods are in an interface.